tests/parametric/test_otel_env_vars.py

```diff
@@ -183,6 +184,7 @@ def test_otel_log_level_to_debug_mapping(self, test_agent, test_library):
     @missing_feature(
         context.library == "ruby", reason="does not support enabling opentelemetry via DD_TRACE_OTEL_ENABLED"
     )
+    @irrelevant(context.library == "golang", reason="does not support enabling opentelemetry via DD_TRACE_OTEL_ENABLED")
```

Comment:
The test below is skipped with the same reason but with `missing_feature`, is it on purpose ? 

Reminder : 

* `missing_feature` : the feature is not done, but will be done one day. Must be used even if it's not prioritized
* `irrelevant` : the feature can't exists for technical or design choice reasons. This decorator can also be used if the feature won't be implemented for 

The main difference between those two decorator is that system-tests don't  won't bother you with a `XPASS` warning in the CI if one day, a `irrelevant` feature fulfil the test.

Reply:
Given the definitions above + the current implementation/limitations of dd-trace-go, I would consider this feature irrelevant.